CHANGES.rst

```diff
@@ -9,8 +9,8 @@ Unreleased
 Version 8.0.2
 -------------
 
-Unreleased
-
+-   An an argument to ``Path`` which mirrors writable/readable, but for
```

Comment:
This needs to go in the 8.1 section.

Reply:
Thanks, wasn't sure whether this could be minor or not.